client/my-sites/customer-home/cards/tasks/index.jsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { Button } from '@automattic/components';
+import { isMobile } from '@automattic/viewport';
+
+/**
+ * Internal dependencies
+ */
+import ActionPanel from 'components/action-panel';
+import ActionPanelTitle from 'components/action-panel/title';
+import ActionPanelBody from 'components/action-panel/body';
+import ActionPanelFigure from 'components/action-panel/figure';
+import ActionPanelCta from 'components/action-panel/cta';
+import Gridicon from 'components/gridicon';
+import QueryPublicizeConnections from 'components/data/query-publicize-connections';
+import { savePreference } from 'state/preferences/actions';
+import { getPreference } from 'state/preferences/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { getConnectionsBySiteId } from 'state/sharing/publicize/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class RecommendedTask extends Component {
```

Comment:
Let's maybe split this out to focus on the single connection task. We can build a recommended task container (like primary) that helps renders tasks noted from the API. We can build this out in a follow up PR.

Overall, I think it'd be fine to rename this from RecommendedTask to some like `ConnectAccountsTask` or `DriveTrafficTask`.

Here's the latest mock from @danhauk though we might iterate on this further:

<img width="1024" alt="growth-task" src="https://user-images.githubusercontent.com/1270189/79023808-0f197b00-7b36-11ea-8bed-cd5461594ad8.png">

Reply:
I added a commit to split the tasks into a single one and changed the title to match the new one. :)